content/en/tutorials/securely_accessing_network_locally.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Securely Accessing Opta Networks Locally"
+linkTitle: "Securely Accessing Opta Networks Locally"
+date: 2022-01-03
+draft: false
+description: >
+  Instructions to integrate with the CI/CD platform of your choice
+---
+
+# Overview
+When working with secure private networks, and specifically with the databases in them, one may wish to somehow manually 
+connect to them for debugging purpose, but there is This action, naturally, has security concerns and there's many 
+solutions out there. For our users, we typically recommend installing VPN servers within your opta-managed VPCs and 
+handling network access to your environments from there. For just getting started, however, there's a rather 
```

Comment:
for `getting started` do we mean for development only?
(aka not for production use)

Reply:
done